source/JsMaterialX/JsMaterialXCore/JsTypes.cpp

```diff
@@ -113,6 +108,9 @@ extern "C"
         ems::function("DEFAULT_TYPE_STRING", ems::optional_override([]() {
                      return mx::DEFAULT_TYPE_STRING;
                  }));
+        ems::function("EMPTY_STRING", ems::optional_override([]() {
```

Comment:
Any idea why these constants are exposed as functions and not via
https://emscripten.org/docs/api_reference/bind.h.html?highlight=bind%20h#constants
?

Reply:
Nope. I've put it on the todo list.